packages/twenty-server/src/engine/api/graphql/graphql-query-runner/utils/compute-where-condition-parts.ts

```diff
@@ -96,28 +95,34 @@ export const computeWhereConditionParts = ({
         params: { [`${key}${uuid}`]: `${value}` },
       };
     case 'contains':
-      if (isTSVectorField) {
-        const tsQuery = value
-          .split(/\s+/)
-          .map((term: string) => `${term}:*`)
-          .join(' & ');
-
-        return {
-          sql: `(
-            "${objectNameSingular}"."${key}" @@ to_tsquery('simple', :${key}${uuid}Ts) OR
-            "${objectNameSingular}"."${key}"::text ILIKE :${key}${uuid}Like
-          )`,
-          params: {
-            [`${key}${uuid}Ts`]: tsQuery,
-            [`${key}${uuid}Like`]: `%${value}%`,
-          },
-        };
-      }
-
       return {
         sql: `"${objectNameSingular}"."${key}" @> ARRAY[:...${key}${uuid}]`,
         params: { [`${key}${uuid}`]: value },
       };
+    case 'search': {
+      if (!isTSVectorField) {
```

Comment:
It feels appropriate to have a dedicated operator but the check here probably isn't required as we would also need to check for all other field types (e.g. like might not be implemented on numbers etc), so validation does not belong here